requirements.txt

```diff
@@ -1,6 +1,7 @@
 h5py
 ismrmrd
 numpy
+matplotlib
```

Comment:
Not sure why we need this additional dependency, can you explain?

Otherwise you can just commit the suggestion
```suggestion
```

Reply:
Holy sorry I did this by mistake